kinds/dashboard/dashboard_kind.cue

```diff
@@ -221,6 +221,8 @@ lineage: seqs: [
 					// TODO docs
 					color: string @grafanamaturity(NeedsExpertReview)
 					// TODO docs
+					index?: int32 @grafanamaturity(NeedsExpertReview)
```

Comment:
Apparently this is an expected field here, but we just never had it? Either that, or the thresholds logic needs to be updated in order to remove what is ephemeral runtime state.